package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "cql-exec-fhir-mongo",
+  "version": "1.0.0",
+  "description": "Provides a FHIR data provider for use with CQL",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint \"**/*.js\"",
+    "lint:fix": "eslint \"**/*.js\" --fix",
+    "prettier": "prettier --check \"**/*.js\"",
+    "prettier:fix": "prettier --write \"**/*.js\"",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
CI checks fail here because we have no test script. Wanna just add a `test/fhir.test.js` file that has one test that always passes? Then we can add more tests later.

You can also bring in jest as a devDependency as part of this.